src/prettytable/prettytable.py

```diff
@@ -2197,11 +2231,16 @@ def _get_simple_html_string(self, options):
             for field in self._field_names:
                 if options["fields"] and field not in options["fields"]:
                     continue
-                lines.append(
-                    "            <th>{}</th>".format(
-                        escape(field).replace("\n", linebreak)
+                if options["escape_header"]:
+                    lines.append(
+                        "            <th>{}</th>".format(
+                            escape(field).replace("\n", linebreak)
+                        )
+                    )
+                else:
+                    lines.append(
+                        "            <th>{}</th>".format(field.replace("\n", linebreak))
                     )
```

Comment:
Can we simplify this?
```suggestion

                if options["escape_header"]:
                    field = escape(field)

                lines.append(
                    "            <th>{}</th>".format(field.replace("\n", linebreak))
                )
```


But with this version, or the current PR version, the tests pass even if we replace `escape(field)` with `field`. Please can you check this is tested properly?

Reply:
I added two new tests that checks the current escaped output